.circleci/docker/centos-rocm/Dockerfile

```diff
@@ -74,6 +74,7 @@ RUN rm install_rocm.sh
 COPY ./common/install_rocm_magma.sh install_rocm_magma.sh
 RUN bash ./install_rocm_magma.sh
 RUN rm install_rocm_magma.sh
+ENV ROCM_PATH /opt/rocm
```

Comment:
Needed due to following: Once `/opt/rocm/lib/cmake/hip/FindHIP.cmake` set HIP_ROOT_DIR, ROCM_PATH is required in the ENV. https://gerrit-git.amd.com/c/compute/ec/hip/+/903237 still not merged into mainline. Also, rocm5.7_internal_testing/upstream_main branches have ENV ROCM_PATH set for triton builds, so this is in alignment